src/parser/nodes/IfNode.java

```diff
@@ -0,0 +1,61 @@
+package parser.nodes;
+
+import java.util.ArrayList;
+
+import pacman.game.Game;
+import parser.Condition;
+import parser.operators.BooleanOperator;
+
+public class IfNode extends Node {
+	private ArrayList<Condition> conditionList;
+	private ArrayList<BooleanOperator> boolOpList;
+	
+	public IfNode(){
+		super();
+		conditionList = new ArrayList<>();
+		boolOpList = new ArrayList<>();
+	}
+	
+	public IfNode(ArrayList<Condition> cl, ArrayList<BooleanOperator> bopl){
+		super();
+		conditionList = cl;
+		boolOpList = bopl;
+	}
+	
+	public Node execute(Game g){
+		boolean ev = this.evaluateIfNode(g);
+		if(ev)
+			return this.children.get(0).execute(g);
+		else if (!ev && this.children.size() > 1)
```

Comment:
Hacer una función booleana que se llame `isIfElse()` y que compruebe lo del size, por claridad. `this.isIfElse()` ftw